docs/source/user/inflowwind/driver.rst

```diff
@@ -137,4 +138,100 @@ When converting from a full-field wind format to a uniform wind file, the follow
 - The power law exponent is either
   
   1. The power-law exponent specified in InflowWind (if a power law wind profile is used to add to the turbulence with native-Bladed or HAWC2 files), or
-  2. Calculated by using the mean wind speeds at two points: the reference (hub) height and the uppermost height on the grid.
\ No newline at end of file
+  2. Calculated by using the mean wind speeds at two points: the reference (hub) height and the uppermost height on the grid.
+
+accel flag
+-------------------
+
+The ability to calculate the acceleration of the flow field was added to InflowWind
+to support the analysis of MHK, underwater, turbines. The acceleration is needed
+to calculate the mass effects of the fluid interacting with the rotor. Calculation of the
```

Comment:
Change "mass effects" to "inertia effects" or "fluid-inertia effects.

Reply:
Done!